### PR TITLE
Tests for CASSANDRA-9961

### PR DESCRIPTION
**Pull request by stef1927**
I created two tests for CASSANDRA-9961, to describe materialized views in cqlsh.

### CODE REVIEW

**Comment by stef1927** [comment]
Actually I converted one of the two tests to cover CASSANDRA-10348 as well.

File: cqlsh_tests/cqlsh_tests.py [hunk 1]
```diff
@@ -1267,6 +1318,54 @@ def test_describe_round_trip(self):
         # the table created before and after should be the same
         self.assertEqual(reloaded_describe_out, describe_out)
 
+    @require("10348")
+    @since('3.0')
+    def test_materialized_view(self):
+        """
+        Test operations on a materialized view: create, describe, select from, drop, create using describe output.
+        @jira_ticket CASSANDRA-9961 and CASSANDRA-10348
```
Comment (mambocab) [inline cqlsh_tests/cqlsh_tests.py]: Will the doxygen tooling handle this, or does this need to be formatted some other way? /cc @kishkaru

**Comment by mambocab** [comment]
These are great, thank you. Once @kishkaru drops some Doxygen knowledge on us, I'll be +1.

File: cqlsh_tests/cqlsh_tests.py (hunk 1 above)
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_tests.py]: Doxygen will be able to handle this.

**Comment by mambocab** [comment]
Never mind, Philip says it'll work. LGTM, merge away.

**Comment by stef1927** [comment]
I've added one more test for CASSANDRA-10328 as well as more test cases for CASSANDRA-9961, which were requested during code review. Both CASSANDRA-10348 and CASSANDRA-10328 will be closed as duplicates of 9961. 

Please review the latest commit as well, thanks!

File: cqlsh_tests/cqlsh_tests.py [hunk 2]
```diff
@@ -692,6 +692,38 @@ def test_describe_on_non_reserved_keywords(self):
...
+        self.assertIn("users_by_state", output)
+
+        self.execute(cql="DESCRIBE MATERIALIZED VIEW test.users_by_state", expected_output=self.get_users_by_state_mv_output())
+        self.execute(cql="DESCRIBE test.users_by_state", expected_output=self.get_users_by_state_mv_output())
+        self.execute(cql="USE test; DESCRIBE MATERIALIZED VIEW test.users_by_state", expected_output=self.get_users_by_state_mv_output())
+        self.execute(cql="USE test; DESCRIBE MATERIALIZED VIEW users_by_state", expected_output=self.get_users_by_state_mv_output())
+        self.execute(cql="USE test; DESCRIBE users_by_state", expected_output=self.get_users_by_state_mv_output())
+
+        # test quotes
+        self.execute(cql="USE test; DESCRIBE MATERIALIZED VIEW \"users_by_state\"", expected_output=self.get_users_by_state_mv_output())
```
Comment (mambocab) [inline cqlsh_tests/cqlsh_tests.py]: You can avoid escaping double-quotes by using `'` as the delimiter for these strings. In Python `'` and `"` are identical.

**Comment by mambocab** [comment]
Both new commits look good to me; if you like you can clean up one of the CQL strings as I mentioned in my comment. +1, thanks!

**Comment by stef1927** [comment]
I cleaned up the quotes and squashed, thanks. You can merge now or wait for CASSANDRA-9961 to be committed (it should happen soon) and merge without the require tags.

**Comment by mambocab** [comment]
Sorry for the delayed review. The most recent commit looks great, thank you!

**Comment by stef1927** [comment]
Un-required and merged, thank you.

**Pull request merged**